packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.routing.js

```diff
@@ -4,13 +4,28 @@ export default /* @ngInject */ ($stateProvider) => {
     views: {
       kubernetesView: 'pciProjectKubernetesDetailsAuditLogs',
     },
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('isFeatureAvailable')
+        .then((isFeatureAvailable) =>
+          isFeatureAvailable
+            ? false
+            : { state: 'pci.projects.project.kubernetes.details' },
+        ),
     resolve: {
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('kube_logs_tab_title'),
       logKindsList: /* @ngInject */ (Kubernetes, projectId) =>
         Kubernetes.getKubeLogKinds(projectId),
       kind: /* @ngInject */ (logKindsList, $state) =>
         $state.params.kind || logKindsList[0].name,
+      isFeatureAvailable: /* @ngInject */ (ovhFeatureFlipping) =>
+        ovhFeatureFlipping
+          .checkFeatureAvailability('public-cloud:log-to-customer')
+          .then((feature) =>
+            feature.isFeatureAvailable('public-cloud:log-to-customer'),
+          ),
```

Comment:
You have already isLogToCustomerFeatureAvailable on parent. I think it would be available on child for avoid a new 2api call.